src_updater/updater.py

```diff
@@ -172,7 +172,7 @@ def handle_error(self):
 
 
     def open_report_page(self):
-        webbrowser.open("https://github.com/Hyphen-ated/RebirthItemTracker/issues", autoraise=True)
+        webbrowser.open("https://github.com/rchardon/RebirthItemTracker/issues", autoraise=True)
```

Comment:
In a pull request, you shouldn't change references pointing to the main repository. This cannot be merged while this is present.